src/components/Board/Board.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Row from '../Row/Row';
+import HorizontalWallRow from '../HorizontalWallRow/HorizontalWallRow';
+import GameLogic from '../../hooks/gameLogic';
+
+function Board() {
```

Comment:
import React from 'react';
import Row from '../Row/Row';
import HorizontalWallRow from '../HorizontalWallRow/HorizontalWallRow';
import GameLogic from '../../hooks/gameLogic';

const Board = () => {
    const boardSize = 9;
    const { state, handlePlayerClick, movePlayer, handleWallHover } = GameLogic(boardSize);
    const { highlightedSquares, players, hoveredWalls } = state;

    const renderRow = (rowIndex) => (
        <Row
            key={`row-${rowIndex}`}
            id={`row-${rowIndex}`}
            boardSize={boardSize}
            rowIndex={rowIndex}
            highlightedSquares={highlightedSquares}
            onPlayerClick={handlePlayerClick}
            movePlayer={movePlayer}
            players={players}
            onWallHover={handleWallHover}
            hoveredWalls={hoveredWalls}
        />
    );

    const renderHorizontalWallRow = (rowIndex) => (
        <HorizontalWallRow
            key={`hrow-${rowIndex}`}
            id={`hrow-${rowIndex}`}
            rowIndex={rowIndex}
            onWallHover={handleWallHover}
            hoveredWalls={hoveredWalls}
        />
    );

    const renderBoard = () => {
        const rows = [];
        for (let i = 0; i < boardSize; i++) {
            rows.push(renderRow(i));
            if (i !== boardSize - 1) {
                rows.push(renderHorizontalWallRow(i));
            }
        }
        return rows;
    };

    return (
        <table className='Board'>
            <tbody>{renderBoard()}</tbody>
        </table>
    );
};

export default Board;